packages/shorebird_cli/test/src/commands/patch/patch_android_command_test.dart

```diff
@@ -270,7 +270,7 @@ flutter:
       when(() => argResults['staging']).thenReturn(false);
       when(() => argResults['dry-run']).thenReturn(false);
       when(() => argResults['force']).thenReturn(false);
-      when(() => argResults['release-version']).thenReturn(release.version);
+      when(() => argResults['release-version']).thenReturn(null);
```

Comment:
I think we can just delete this line since returning `null` is the default behavior.

Reply:
Good call, done